docs/HowToUpdate.md

```diff
@@ -0,0 +1,65 @@
+# How to Update
+
+This document outlines how to update parts of the website. See `Contributing.md` to see how to contribute to the repo. 
```

Comment:
make `Contributing.md` a link to the file and not a code snippet

Reply:
Fixed